src/modals/metadataData/EditMetadataDataModal.vue

```diff
@@ -0,0 +1,235 @@
+<script setup>
+import { store } from '../../store.js'
+</script>
+<template>
+	<NcModal
+		v-if="store.modal === 'editMetadataDataModal'"
+		ref="modalRef"
+		@close="store.setModal(false)">
+		<div class="modal__content">
+			<h2>Edit Metadata property {{ store.metadataDataKey }}</h2>
+
+			<div v-if="success === -1">
+				<div v-if="!loading" class="form-group">
+					<NcSelect v-bind="typeOptions"
+						v-model="metadata.properties[store.metadataDataKey].type"
+						required
+						:loading="metadataLoading" />
+
+					<NcTextField :disabled="loading"
+						label="description"
+						:value.sync="metadata.properties[store.metadataDataKey].description"
+						:loading="metadataLoading" />
+
+					<NcTextField :disabled="loading"
+						label="format"
+						:value.sync="metadata.properties[store.metadataDataKey].format"
+						:loading="metadataLoading" />
+
+					<NcTextField :disabled="loading"
+						label="max date"
+						:value.sync="metadata.properties[store.metadataDataKey].maxDate"
+						:loading="metadataLoading" />
+
+					<NcTextField :disabled="loading"
+						label="reference"
+						:value.sync="metadata.properties[store.metadataDataKey].$ref"
+						:loading="metadataLoading" />
+
+					<NcCheckboxRadioSwitch
+						:checked.sync="metadata.properties[store.metadataDataKey].required">
+						Required
+					</NcCheckboxRadioSwitch>
+
+					<NcCheckboxRadioSwitch
+						:checked.sync="metadata.properties[store.metadataDataKey].default">
+						Default
+					</NcCheckboxRadioSwitch>
+
+					<NcCheckboxRadioSwitch
+						:checked.sync="metadata.properties[store.metadataDataKey].cascadeDelete">
+						Cascade delete
+					</NcCheckboxRadioSwitch>
+
+					<NcTextField :disabled="loading"
+						type="number"
+						label="Exclusive minimum"
+						:value.sync="metadata.properties[store.metadataDataKey].exclusiveMinimum"
+						:loading="metadataLoading" />
+				</div>
+			</div>
+
+			<NcLoadingIcon

```

Comment:
Je zet hier nu de nc loading ipv het formulier, maar we zijn in de breedte over gestapt naar het patroon alle velden dan op inactief te zetten en een loading icoon in de toevoegen knop te zetten. Dat is grafisch wat subtieler.